pkg/conf/testdata/testConfig.yml

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
Might be better to update the current testdata/testConfig.yml in doctor instead of creating another. Two config files is more work to keep in sync than one and the only change with this one is port_range or refactor both to use the same

Reply:
done